graph_creator/models.py

```diff
@@ -3,23 +3,124 @@
 """
 
 from __future__ import unicode_literals
-from datetime import datetime
+import datetime
+import json
 
 from django.db import models
+from django.db.models import Sum, Count, DateField
+from django.db.models.functions import Trunc
+
+
+def get_previous_day_start_and_end_dates():
+    current_datetime = datetime.datetime.today() - datetime.timedelta(days=1)
+    start_of_day = current_datetime.replace(hour=0, minute=0, second=0, microsecond=0)
+    end_of_day = current_datetime.replace(hour=23, minute=59, second=59, microsecond=59)
+
+    return start_of_day, end_of_day
 
 
 class DataStorage(models.Model):
     """
     Model that stores data received from the edx-platform.
     """
 
-    active_students_amount = models.IntegerField(default=0)
+    active_students_amount_day = models.IntegerField(default=0)
+    active_students_amount_week = models.IntegerField(default=0)
+    active_students_amount_month = models.IntegerField(default=0)
     courses_amount = models.IntegerField(default=0)
-    last_data_update = models.DateTimeField(default=datetime.now())
+    data_update = models.DateTimeField(default=datetime.datetime.now())
     latitude = models.FloatField(default=0, blank=True)
     longitude = models.FloatField(default=0, blank=True)
     platform_name = models.CharField(max_length=255, null=True, blank=True)
     platform_url = models.URLField(null=True, blank=True)
     secret_token = models.CharField(max_length=255, null=True)
     statistics_level = models.CharField(max_length=255)
     students_per_country = models.TextField()
```

Comment:
We save there string, that contains json-oriented we will `loads()`. 
Example: '{"RU": 1200, "AS": 1300, "UA": 1400, "null": 1}'.